Xamarin.Kotlin.StdLib version works from (>= 1.5.31.1)

Hi,

As part of the [conversation](https://github.com/jedusei/Twilio.Conversations.Xamarin/issues/3), I forgot to mention. 

One of the Twilio.Conversations.Xamarin 's dependences, [Xamarin.Kotlin.StdLib] (>= 1.5.31) 
 will be trouble to initialise IConversationClient. 

I have upgraded to [Xamarin.Kotlin.StdLib] (>= 1.5.31.1) then it became working fine. It might be good to update to version description in the dependency section in [nuget package page](https://www.nuget.org/packages/Twilio.Conversations.Xamarin/#dependencies-body-tab). 

Or I will do if I know how to do that :)

Hi @jedusei 

Yes. It causes at `IConversationsClient.CreateAsync(context, token, properties);`

The error was 
```
`{Java.Lang.NoClassDefFoundError: Failed resolution of: Lkotlin/jvm/JvmClassMappingKt; ---> Java.Lang.ClassNotFoundException: Didn't find class "kotlin.jvm.JvmClassMappingKt" on path: DexPathList[[zip file "/data/app/com.companyname.app1-1/base.apk"],nativeLibraryDirectories=[/data/app/com.companyname.app1-1/lib/x86_64, /vendor/lib64, /system/lib64]]`
```

The stack trace is below. Sorry for ugly text.
```
--- End of inner exception stack trace ---
  at Java.Interop.JniEnvironment+StaticMethods.CallStaticObjectMethod (Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00068] in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/Java.Interop/JniEnvironment.g.cs:12890 
  at Android.Runtime.JNIEnv.FindClass (System.String classname) [0x0004c] in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNIEnv.cs:572 
  at Twilio.Conversations.IConversationsClient..cctor () [0x00000] in C:\Users\Joe\source\repos\Twilio.Conversations.Xamarin\Twilio.Conversations.Droid\Additions\Fixes\IConversationsClient.cs:14 
  --- End of managed Java.Lang.NoClassDefFoundError stack trace ---
java.lang.NoClassDefFoundError: Failed resolution of: Lkotlin/jvm/JvmClassMappingKt;
	at com.twilio.conversations.ConversationsClientImpl.<clinit>(ConversationsClientImpl.java:66)
	at java.lang.Class.classForName(Native Method)
	at java.lang.Class.forName(Class.java:309)
	at crc64a7a6b04b89628087.MainActivity.n_onCreate(Native Method)
	at crc64a7a6b04b89628087.MainActivity.onCreate(MainActivity.java:37)
	at android.app.Activity.performCreate(Activity.java:5990)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2278)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
	at android.app.ActivityThread.access$800(ActivityThread.java:151)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5254)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
Caused by: java.lang.ClassNotFoundException: Didn't find class "kotlin.jvm.JvmClassMappingKt" on path: DexPathList[[zip file "/data/app/com.companyname.app1-1/base.apk"],nativeLibraryDirectories=[/data/app/com.companyname.app1-1/lib/x86_64, /vendor/lib64, /system/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
	... 18 more
	Suppressed: java.lang.ClassNotFoundException: kotlin.jvm.JvmClassMappingKt
		at java.lang.Class.classForName(Native Method)
		at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
		at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
		... 19 more
	Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack available
}
```

--- a/Twilio.Conversations.Droid/Twilio.Conversations.Droid.csproj
+++ b/Twilio.Conversations.Droid/Twilio.Conversations.Droid.csproj
@@ -114,7 +114,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Xamarin.Kotlin.StdLib">
-      <Version>1.5.31</Version>
+      <Version>1.5.31.4</Version>
     </PackageReference>
     <PackageReference Include="Xamarin.Kotlin.StdLib.Jdk8">
       <Version>1.5.0</Version>

--- a/Twilio.Conversations.Xamarin.nuspec
+++ b/Twilio.Conversations.Xamarin.nuspec
@@ -13,7 +13,7 @@
 		<tags>twilio conversations xamarin ios android</tags>
 		<dependencies> 
 			<group targetFramework="MonoAndroid44">
-				<dependency id="Xamarin.Kotlin.StdLib" version="1.5.31" />
+				<dependency id="Xamarin.Kotlin.StdLib" version="1.5.31.4" />
 				<dependency id="Xamarin.Kotlin.StdLib.Jdk8" version="1.5.0" />
 				<dependency id="Xamarin.ReLinker" version="1.4.4" />
 				<dependency id="Square.OkHttp3" version="4.9.1" />

